File: web/src/test/java/top/bettercode/summer/web/config/WebMvcConfigurationTest.java
package top.bettercode.summer.web.config;

import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;
import top.bettercode.summer.test.BaseWebNoAuthTest;

/**
 * @author Peter Wu
 */
class WebMvcConfigurationTest extends BaseWebNoAuthTest {


  @Disabled
  @Test
  public void test() throws Exception {
    download(get("/captcha.jpg"), "captcha.jpg");
  }

}

File: web/src/test/kotlin/top/bettercode/summer/test/web/config/WebMvcConfigurationTest.kt
package top.bettercode.summer.test.web.config

import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test
import top.bettercode.summer.test.BaseWebNoAuthTest

/**
 * @author Peter Wu
 */
internal class WebMvcConfigurationTest : BaseWebNoAuthTest() {
    @Disabled
    @Test
    fun test() {
        download(get("/captcha.jpg"), "captcha.jpg")
    }
}